test/backup.ts

```diff
@@ -322,6 +323,114 @@ describe('Backup', () => {
       );
     });
 
+    describe('copy', () => {
+      const INSTANCE_NAME = 'instance-name';
+      const SRC_BACKUP = 'src-backup';
+      const DST_BACKUP = 'dst-backup';
+
+      beforeEach(() => {
+        fakeCodec.encode = util.noop;
+        backup = new Backup(INSTANCE, DST_BACKUP, SRC_BACKUP);
+      });
+
+      it('should succeed with basic params', done => {
```

Comment:
nit: Change test name to `should make the correct request` to keep it consistent.